pkg/kubectl/kubectl.go

```diff
@@ -103,6 +103,7 @@ func (e ShortcutExpander) VersionAndKindForResource(resource string) (defaultVer
 func expandResourceShortcut(resource string) string {
 	shortForms := map[string]string{
 		// Please keep this alphabetized
+		"c":      "components",
```

Comment:
given the not-so-common use of this, can we say "comp" as the shortcut?  We're going to run out of them eventually..